src/pudl/analysis/ferc1_eia_train.py

```diff
@@ -627,13 +659,38 @@ def _add_to_null_overrides(null_matches, current_null_overrides_path) -> None:
     out_null_matches.to_csv(current_null_overrides_path, index=False)
 
 
+def _add_to_one_to_many_overrides(one_to_many, current_one_to_many_path) -> None:
+    """Add record_id_ferc1 values verified to have multiple EIA matches to csv."""
+    logger.info(
+        "Adding record_id_ferc1 values with multiple EIA matches to one_to_many_overrides csv"
+    )
+    current_one_to_many = pd.read_csv(current_one_to_many_path)
+    new_one_to_many = (
+        one_to_many[
+            ["record_id_eia", "record_id_ferc1", "signature_1", "signature_2", "notes"]
+        ]
+        .copy()
+        .drop_duplicates(subset=["record_id_eia", "record_id_ferc1"])
+    )
+    logger.debug(
+        f"Found {len(new_one_to_many.record_id_ferc1.unique())} new plants with multiple EIA matches."
+    )
+    # Combine new and old training data; drop old data in favor or new overrides
+    one_to_many = pd.concat([current_one_to_many, new_one_to_many]).drop_duplicates(
+        subset=["record_id_ferc1", "record_id_eia"], keep="last"
+    )
+    # Write the combined values out to the same location as before
+    one_to_many.to_csv(current_one_to_many_path, index=False)
+
+
 def validate_and_add_to_training(
     utils_eia860,
     ppe,
     ferc1_eia,
     input_dir_path,
     expect_override_overrides=False,
     allow_mismatched_utilities=True,
+    one_to_many=False,
```

Comment:
I'm going to leave this as a parameter with the default set to `True`. I agree that it should be the default behavior, but I'd like to leave this functionality available to toggle given that it's an additional layer of abstraction added onto the existing plant part infrastructure.

Reply:
Ok that makes sense. I wonder if we should null out the EIA matches for FERC records if `one_to_many = False` so that they aren't mapped rather than being mapped to just one of their EIA subcomponents.